src/render.c

```diff
@@ -1174,6 +1341,14 @@ bool init_render(session_t *ps) {
 			return false;
 		}
 	}
+
+	// Initialize our rounded corners fragment shader
+	if (ps->o.corner_radius > 0) {
```

Comment:
The segfault appears to occur because you create the gl shader for rounded corners even if the glx backend is not being used.

I suggest that these lines should be amended to something more like this:

```
if (ps->o.corner_radius > 0 && ps->o.backend == BKEND_GLX) {
#ifdef CONFIG_OPENGL
    if (!glx_init_rounded_corners(ps)) {
        log_error("Failed to init rounded corners shader.");
        return false
    }
#else
    assert(false);
#endif
}
```

Reply:
Thank you @sdhand, I didn't have the time to touch the code again yet but this is definitely needed, it's very possible I missed other places where I should have added  `#ifdef CONFIG_OPENGL` it was on my TODO list.

I just committed your fix into commit aaf337e.